examples/tiled.ipynb

```diff
@@ -0,0 +1,165 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Demonstration of Tiled Functionality\n",
+    "This notebook demonstrates how to connect to tiled using the python client. \n",
+    "\n",
+    "Not that config.py is in gitignore because it contains the api_key. It should look like:\n",
+    "\n",
+    "```\n",
+    "api_key=\"<key>\"\n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import tempfile  used for on_disk cache\n",
+    "import ipywidgets as widgets\n",
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib.animation as animation \n",
+    "from tiled.client import from_uri\n",
+    "from tiled.client.cache import Cache\n",
+    "# from tiled.client import show_logs\n",
+    "import config\n",
+    "\n",
+    "%matplotlib widget\n",
+    "\n",
+    "# displays details of tiled client http traffic\n",
+    "# it's a little wordy but often extremely useful\n",
```

Comment:
May be worth to split here to have the import package cell separate from setting up the client.